WordPress/Classes/ViewRelated/Reusable SwiftUI Views/ButtonStyles.swift

```diff
@@ -1,19 +1,14 @@
 import SwiftUI
 
-struct PrimaryButtonStyle: ViewModifier {
-    func body(content: Content) -> some View {
-        content
-            .font(.headline)
-            .padding()
-            .frame(minWidth: 0, maxWidth: .infinity, minHeight: 44.0, maxHeight: 44.0)
-            .background(Color(.primary))
-            .foregroundColor(.white)
-            .clipShape(RoundedRectangle(cornerRadius: 8.0))
-    }
-}
+struct PrimaryButtonStyle: ButtonStyle {
```

Comment:
According to Apple, this is the recommended way to define button styles in SwiftUI. See [documentation](https://developer.apple.com/documentation/swiftui/buttonstyle).

Also, this type was never used before, so refactoring it won't cause any regression.